dotcom-rendering/src/web/components/MostViewedFooterLayout.tsx

```diff
@@ -16,30 +18,39 @@ const stackBelow = (breakpoint: Breakpoint) => css`
 	}
 `;
 
-const fixedWidths = css`
+const fixedWidths = (
+	renderAds: boolean | undefined,
+	isFront: boolean | undefined,
+) => css`
 	width: 100%;
 	${between.desktop.and.wide} {
 		min-width: 627px;
 	}
 	${from.wide} {
 		min-width: 718px;
 	}
+	${!renderAds && `${from.leftCol}{width: 75%;}`}
+	${isFront && `${between.leftCol.and.wide}{margin-top: -34px;}`}
 `;
 
-export const MostViewedFooterLayout = ({ children }: Props) => {
+export const MostViewedFooterLayout = ({
+	renderAds,
+	isFront,
+	children,
+}: Props) => {
 	return (
 		<div
 			data-print-layout="hide"
 			className="content-footer"
 			css={stackBelow('desktop')}
 		>
-			<div css={fixedWidths}>{children}</div>
+			<div css={fixedWidths(renderAds, isFront)}>{children}</div>
```

Comment:
```suggestion
			<div css={[fixedWidths, renderAds && adStyles, isFront && frontStyles]}>{children}</div>
```

Instead of making `fixedWidth` a function could we instead just have seperate styles for ads and fronts that we apply? I think Emotion has a better time at optimizing the CSS if its done this way.